mm-mock-server/src/mocks/queries.ts

```diff
@@ -18,12 +18,17 @@ export function mockQueries(serverState: MockServerState) {
                 generators.generateChannel([serverState.loggedInUser, serverState.otherUsers[0]]),
             ]
         },
-        findUsers: () => {
+        findChannelInvitationsForUser: () => {
             return [
-                serverState.loggedInUser,
-                serverState.otherUsers[0],
+                generators.generateChannelInvitation(serverState.loggedInUser, serverState.otherUsers[0], false, false),
+                generators.generateChannelInvitation(serverState.loggedInUser, serverState.otherUsers[0], false, true),
+                generators.generateChannelInvitation(serverState.otherUsers[0], serverState.loggedInUser, false, false),
+                generators.generateChannelInvitation(serverState.otherUsers[0], serverState.loggedInUser, false, true),
             ]
         },
+        findUsers: () => {
+            return serverState.otherUsers.concat([serverState.loggedInUser]);
+        },
         getAuthenticatedUser: () => {
             if (!serverState.loggedIn) {
                 return null;
```

Comment:
This doesn't match the signature in the backend that we've implemented. If a logged out user runs `getAuthenticatedUser`, there is an error rather than a null response.

@shadowbrush this is [a question Miguel had](https://trello.com/c/KeSnySPu/49-servererror-when-launching-app-without-being-logged-in) for us yesterday: do we expose any way on the backend to check whether a user is logged in without catching an error? At the moment, I didn't see any, and it feels like we can probably avoid running into this error by checking the logged in state on the front end, and only fetching a user if the front end "thinks" that the user is logged in.

Reply:
I responded to the Trello card, Chris. I'd say we can leave it as it is right now, but the backend needs to throw a better exception.